translator/trace/jaegerthrift_to_protospan_test.go

```diff
@@ -217,6 +231,20 @@ func TestConservativeConversions(t *testing.T) {
 						Code:    13,
 						Message: "proxy crashed",
 					},
+					Attributes: &tracepb.Span_Attributes{
+						AttributeMap: map[string]*tracepb.AttributeValue{
+							"status.code": {
```

Comment:
I don't think we want these attributes as they are, they should have been translated in status.